src/scripts/main.js

```diff
@@ -1,3 +1,17 @@
 'use strict';
 
-// write your code here
+const allCountryPopulations = [...document.querySelectorAll('.population')]
+  .map(countrypopulation => countrypopulation.innerHTML);
+const sumOfPopulationValues = allCountryPopulations
+  .reduce((accumulator, currentPopulation) => {
+    const currentPopulationValue = currentPopulation.split(',').join('');
+
+    return accumulator + +currentPopulationValue;
```

Comment:
Використовуй явне приведення типів
```suggestion
    return accumulator + Number(currentPopulationValue);
```